src/desktop/components/main_layout/header/index.styl

```diff
@@ -24,8 +24,8 @@ html[data-hide-hero-units]
         visibility visible
         opacity 1
         transition visibility 0s linear 0s, opacity 300ms
-    .avant-garde-button-black
-      display none
+      .avant-garde-button-black
```

Comment:
This should have been indented here (we want to hide the button the in header search bar). Accidentally was not indented enough.

We were developing this with a lab feature (so only for signed up users), and then when we switched from lab -> AB test, we must not have caught this in staging/QA.